### PR TITLE
fix: fill in core.SubTaskMeta.DomainTypes

### LINKED ISSUE #2505: [Bug][Plugins] Some subTask did not specify `DomainTypes`
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

Some subTask's DomainTypes were not set correctly.

### What you expected to happen

All subTask should be set DomainTypes correctly.

### How to reproduce

read the code

### Anything else

_No response_

### Version

main

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by mindlesscloud**
# Summary

fix #2505 [Bug][Plugins] Some subTask did not specify DomainTypes

### Does this close any open issues?
close #2505

### CODE REVIEW

File: plugins/gitee/tasks/commit_extractor.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ var ExtractCommitsMeta = core.SubTaskMeta{
 	EntryPoint:       ExtractApiCommits,
 	EnabledByDefault: true,
 	Description:      "Extract raw commit data into tool layer table GiteeCommit,GiteeAccount and GiteeRepoCommit",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CODE, core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/gitee/tasks/commit_extractor.go:33]: Why it has 2 domains?

File: plugins/gitextractor/tasks/git_repo_collector.go [hunk 2]
```diff
@@ -98,18 +98,21 @@ var CollectGitCommitMeta = core.SubTaskMeta{
 	EntryPoint:       CollectGitCommits,
 	EnabledByDefault: true,
 	Description:      "collect git commits into Domain Layer Tables",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CODE, core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/gitextractor/tasks/git_repo_collector.go:101]: Why does it have 2 domains?

File: plugins/github/tasks/comment_collector.go [hunk 3]
```diff
@@ -130,5 +130,5 @@ var CollectApiCommentsMeta = core.SubTaskMeta{
 	EntryPoint:       CollectApiComments,
 	EnabledByDefault: true,
 	Description:      "Collect comments data from Github api",
-	DomainTypes:      []string{core.DOMAIN_TYPE_CODE, core.DOMAIN_TYPE_TICKET},
+	DomainTypes:      []string{core.DOMAIN_TYPE_CODE_REVIEW, core.DOMAIN_TYPE_TICKET},
```
Comment (Startrekzky) [inline plugins/github/tasks/comment_collector.go:133]: Is this the collector for issue comments or PR comments?

File: plugins/github/tasks/pr_review_extractor.go [hunk 4]
```diff
@@ -31,7 +31,7 @@ var ExtractApiPullRequestReviewersMeta = core.SubTaskMeta{
 	EntryPoint:       ExtractApiPullRequestReviewers,
 	EnabledByDefault: true,
 	Description:      "Extract raw PullRequestReviewers data into tool layer table github_reviewers",
-	DomainTypes:      []string{core.DOMAIN_TYPE_CODE},
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/github/tasks/pr_review_extractor.go:34]: code review?

File: plugins/jira/tasks/issue_changelog_extractor.go [hunk 5]
```diff
@@ -33,7 +33,7 @@ var ExtractIssueChangelogsMeta = core.SubTaskMeta{
 	EntryPoint:       ExtractIssueChangelogs,
 	EnabledByDefault: true,
 	Description:      "extract Jira Issue change logs",
-	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET},
+	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET, core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/jira/tasks/issue_changelog_extractor.go:36]: why add cross?

File: plugins/jira/tasks/issue_collector.go [hunk 6]
```diff
@@ -46,7 +46,7 @@ var CollectIssuesMeta = core.SubTaskMeta{
 	EntryPoint:       CollectIssues,
 	EnabledByDefault: true,
 	Description:      "collect Jira issues",
-	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET},
+	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET, core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/jira/tasks/issue_collector.go:49]: why add cross?

File: plugins/tapd/tasks/task_custom_fields_collector.go [hunk 7]
```diff
@@ -65,4 +65,5 @@ var CollectTaskCustomFieldsMeta = core.SubTaskMeta{
 	EntryPoint:       CollectTaskCustomFields,
 	EnabledByDefault: true,
 	Description:      "collect Tapd TaskCustomFields",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/tapd/tasks/task_custom_fields_collector.go:68]: ticket?

File: plugins/tapd/tasks/task_custom_fields_extractor.go [hunk 8]
```diff
@@ -32,6 +32,7 @@ var ExtractTaskCustomFieldsMeta = core.SubTaskMeta{
 	EntryPoint:       ExtractTaskCustomFields,
 	EnabledByDefault: true,
 	Description:      "Extract raw company data into tool layer table _tool_tapd_task_custom_fields",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/tapd/tasks/task_custom_fields_extractor.go:35]: ticket?

File: plugins/tapd/tasks/task_extractor.go [hunk 9]
```diff
@@ -35,6 +35,7 @@ var ExtractTaskMeta = core.SubTaskMeta{
 	EntryPoint:       ExtractTasks,
 	EnabledByDefault: true,
 	Description:      "Extract raw workspace data into tool layer table _tool_tapd_iterations",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/tapd/tasks/task_extractor.go:38]: ticket?

File: plugins/tapd/tasks/worklog_collector.go [hunk 10]
```diff
@@ -87,4 +87,5 @@ var CollectWorklogMeta = core.SubTaskMeta{
 	EntryPoint:       CollectWorklogs,
 	EnabledByDefault: true,
 	Description:      "collect Tapd worklogs",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/tapd/tasks/worklog_collector.go:90]: ticket?

File: plugins/tapd/tasks/worklog_converter.go [hunk 11]
```diff
@@ -91,4 +91,5 @@ var ConvertWorklogMeta = core.SubTaskMeta{
 	EntryPoint:       ConvertWorklog,
 	EnabledByDefault: true,
 	Description:      "convert Tapd Worklog",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/tapd/tasks/worklog_converter.go:94]: ticket?

File: plugins/tapd/tasks/worklog_extractor.go [hunk 12]
```diff
@@ -32,6 +32,7 @@ var ExtractWorklogMeta = core.SubTaskMeta{
 	EntryPoint:       ExtractWorklogs,
 	EnabledByDefault: true,
 	Description:      "Extract raw workspace data into tool layer table _tool_tapd_iterations",
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
Comment (Startrekzky) [inline plugins/tapd/tasks/worklog_extractor.go:35]: ticket?

File: plugins/gitee/tasks/commit_extractor.go (hunk 1 above)
Comment (mindlesscloud) [reply plugins/gitee/tasks/commit_extractor.go:33]: the `account` is generated too

File: plugins/gitextractor/tasks/git_repo_collector.go (hunk 2 above)
Comment (mindlesscloud) [reply plugins/gitextractor/tasks/git_repo_collector.go:101]: the `account` is generated too

File: plugins/github/tasks/comment_collector.go (hunk 3 above)
Comment (mindlesscloud) [reply plugins/github/tasks/comment_collector.go:133]: both of them, and accounts

File: plugins/github/tasks/pr_review_collector.go [hunk 13]
```diff
@@ -41,7 +41,7 @@ var CollectApiPullRequestReviewsMeta = core.SubTaskMeta{
 	EntryPoint:       CollectApiPullRequestReviews,
 	EnabledByDefault: true,
 	Description:      "Collect PullRequestReviews data from Github api",
-	DomainTypes:      []string{core.DOMAIN_TYPE_CODE},
+	DomainTypes:      []string{core.DOMAIN_TYPE_CROSS},
```
[the remark this answers is not in the document]
Comment (mindlesscloud) [reply plugins/github/tasks/pr_review_collector.go:44]: No, we just extract the reviewer from PR reviews

File: plugins/github/tasks/pr_review_extractor.go (hunk 4 above)
Comment (mindlesscloud) [reply plugins/github/tasks/pr_review_extractor.go:34]: No, we just extract the reviewer from PR reviews

File: plugins/jira/tasks/issue_changelog_collector.go [hunk 14]
```diff
@@ -40,7 +40,7 @@ var CollectIssueChangelogsMeta = core.SubTaskMeta{
 	EntryPoint:       CollectIssueChangelogs,
 	EnabledByDefault: true,
 	Description:      "collect Jira Issue change logs",
-	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET},
+	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET, core.DOMAIN_TYPE_CROSS},
```
[the remark this answers is not in the document]
Comment (mindlesscloud) [reply plugins/jira/tasks/issue_changelog_collector.go:43]: the entity account might be extracted from changelogs

File: plugins/jira/tasks/issue_changelog_convertor.go [hunk 15]
```diff
@@ -38,7 +38,7 @@ var ConvertIssueChangelogsMeta = core.SubTaskMeta{
 	EntryPoint:       ConvertIssueChangelogs,
 	EnabledByDefault: true,
 	Description:      "convert Jira Issue change logs",
-	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET},
+	DomainTypes:      []string{core.DOMAIN_TYPE_TICKET, core.DOMAIN_TYPE_CROSS},
```
[the remark this answers is not in the document]
Comment (mindlesscloud) [reply plugins/jira/tasks/issue_changelog_convertor.go:41]: the entity account might be extracted from changelogs

File: plugins/jira/tasks/issue_changelog_extractor.go (hunk 5 above)
Comment (mindlesscloud) [reply plugins/jira/tasks/issue_changelog_extractor.go:36]: the entity account might be extracted from changelogs

File: plugins/jira/tasks/issue_collector.go (hunk 6 above)
Comment (mindlesscloud) [reply plugins/jira/tasks/issue_collector.go:49]: the entity account might be extracted from changelogs

File: plugins/tapd/tasks/task_custom_fields_collector.go (hunk 7 above)
Comment (mindlesscloud) [reply plugins/tapd/tasks/task_custom_fields_collector.go]: fixed

File: plugins/tapd/tasks/task_custom_fields_extractor.go (hunk 8 above)
Comment (mindlesscloud) [reply plugins/tapd/tasks/task_custom_fields_extractor.go]: fixed

File: plugins/tapd/tasks/worklog_collector.go (hunk 10 above)
Comment (mindlesscloud) [reply plugins/tapd/tasks/worklog_collector.go]: fixed

File: plugins/tapd/tasks/worklog_converter.go (hunk 11 above)
Comment (mindlesscloud) [reply plugins/tapd/tasks/worklog_converter.go]: fixed

**Review by warren830 — APPROVED** [review]
LGTM

**Pull request merged**